pkgs/servers/search/opensearch/default.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, makeWrapper
+, jre_headless
+, util-linux
+, gnugrep
+, coreutils
+, autoPatchelfHook
+, zlib
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "opensearch";
+  version = "2.3.0";
+
+  src = fetchurl {
+    url = "https://artifacts.opensearch.org/releases/bundle/opensearch/2.3.0/opensearch-2.3.0-linux-x64.tar.gz";
+    sha256 = "1jhhz6f58iz5l920avnp0aaai3lfqpqwymk447bjlbvd2arh0rb9";
```

```suggestion
    hash = "";
```

Can we migrate this to an SRI hash instead? If you build this using an empty hash you should get one.